src/app/zap-templates/zcl/data-model/chip/washer-controls-cluster.xml

```diff
@@ -36,7 +44,8 @@ limitations under the License.
 
     <attribute side="server" code="0x0000" define="SPIN_SPEEDS"               type="ARRAY" entryType="CHAR_STRING"                                writable="false" isNullable="false" optional="true">SpinSpeeds</attribute>
     <attribute side="server" code="0x0001" define="SPIN_SPEED_CURRENT"        type="INT8U"                                min="0x00" max="0x1F"   writable="true"  isNullable="true"  optional="true">SpinSpeedCurrent</attribute>
-    <attribute side="server" code="0x0002" define="NUMBER_OF_RINSES"          type="INT8U"                                                        writable="true"  isNullable="true"  optional="true">NumberOfRinses</attribute>
-    <attribute side="server" code="0x0003" define="MAXRINSES"                 type="INT8U"                                min="0x00" max="0x08"   writable="false"                    optional="true">MaxRinses</attribute>
+    <attribute side="server" code="0x0002" define="NUMBER_OF_RINSES"          type="NumberOfRinsesEnum"                                           writable="true"  isNullable="true"  optional="true">NumberOfRinses</attribute>
+    <attribute side="server" code="0x0003" define="SUPPORTED_RINSES"          type="ARRAY" entryType="NumberOfRinsesEnum"                      writable="true"  isNullable="true"  optional="true">SupportedRinses</attribute>
```

Comment:
I don't see this in the spec.  Is this based on some un-merged PR?  If so, can you please point to that PR so this can be reviewed reasonably?

Reply:
Just updated the description, thought I had it in there already, apologies. The PR is https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7225